app/frontend/src/components/Datepicker.test.tsx

```diff
@@ -0,0 +1,86 @@
+import { cleanup, render, screen, waitFor } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import { CHANGE_DATE, SUBMIT } from "features/constants";
+import mockdate from "mockdate";
+import { useForm } from "react-hook-form";
+
+import wrapper from "../test/hookWrapper";
+import Datepicker from "./Datepicker";
+
+beforeAll(() => {
+  mockdate.set("2021-03-20");
+});
+
+afterAll(() => {
+  mockdate.reset();
+});
+
+const submitForm = jest.fn();
+const Form = () => {
+  const { control, register, handleSubmit } = useForm();
+  const onSubmit = handleSubmit((data) => submitForm(data));
+  return (
+    <form onSubmit={onSubmit}>
+      <Datepicker
+        control={control}
+        error={false}
+        helperText=""
+        id="date-field"
+        inputRef={register}
+        label="Date field"
+        name="datefield"
+      />
+      <input type="submit" name={SUBMIT} />
+    </form>
+  );
+};
+
+describe("DatePicker", () => {
+  beforeEach(() => {
+    submitForm.mockClear();
```

Comment:
Don't think this is needed either as `jest.restoreAllMocks()` is called after each test in the global test configuration.